src/components/settings/AvatarWrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import Avatar from "@mui/material/Avatar";
+import { type AvatarProps } from "@mui/material/Avatar";
+import PersonIcon from "@mui/icons-material/Person";
+
+type AvatarWrapperProps = {
+  src: string;
+  alt: string;
+} & AvatarProps;
+
+const AvatarWrapper = ({ src, alt, ...props }: AvatarWrapperProps) => {
+  if (!src || src === "" || src === "https://picsum.photos/200/128") {
```

Comment:
`!src` will also cover `src === ""`, I guess this is when the user has no picture associated with their (google) account. This works well in that case.

Reply:
Ngl a lot of this was late at night so i don't think i acknowledged the first `!src` after I typed it. But this check is also in case we fail to update or include a user with a image url at any point in the future. will rely on the first !src check